src/main/resources/META-INF/rewrite/javaee-to-quarkus.yml

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright 2024 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.quarkus.quarkus2.JavaEEtoQuarkus2Migration
+displayName: Migrate JavaEE to Quarkus 2
+description: These recipes help with the migration of a JavaEE application using EJBs and Hibernate to Quarkus 2. Additional transformations like JSF, JMS, Quarkus Tests may be necessary.
+recipeList:
+  # Migrate Maven Dependencies
+  - org.openrewrite.quarkus.quarkus2.JavaEEtoQuarkus2MavenDependencyMigration
+  # TODO Migrate Gradle Dependencies
+  # Migrate Code
+  - org.openrewrite.quarkus.quarkus2.JavaEEtoQuarkus2CodeMigration
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.quarkus.quarkus2.JavaEEtoQuarkus2MavenDependencyMigration
+displayName: Migrate JavaEE Maven Dependencies to Quarkus 2
+description: Upgrade Standard JavaEE dependencies to Quarkus 2 dependencies.
```

Comment:
I have a recipe locally that does exactly this (and most part of the dependency handling of this recipe). It adds the Camel Extensions for Quarkus, and the output is closer to what `quarkus init` generates. I can contribute it on a separate PR (but I need https://github.com/openrewrite/rewrite/pull/4006 merged before that).

Reply:
#4006 is now merged if you'd like to open that other PR @ammachado

Reply:
Hi @ammachado Did you have a PR for similar functionality ?